New version: OrdinaryDiffEqDifferentiation v1.5.0

JuliaRegistrator opened the pull request:
- Registering package: OrdinaryDiffEqDifferentiation
- Repository: https://github.com/SciML/OrdinaryDiffEq.jl
- Created by: @ChrisRackauckas
- Version: v1.5.0
- Commit: 532231b1a33c46a632579db47f93a49e1ea36f83
- Reviewed by: @ChrisRackauckas
- Reference: https://github.com/SciML/OrdinaryDiffEq.jl/issues/845#issuecomment-2766736246
- Description: High performance ordinary differential equation (ODE) and differential-algebraic equation (DAE) solvers, including neural ordinary differential equations (neural ODEs) and scientific machine learning (SciML)
<!-- 45a4cd22be5b6a15fc3c6d377732d7afb2c24511611bc091bd1f675c67239b47ede25cf955149ce7c597081598ac31ddc600a97776e2dffd738108151c0748b133b3b77a9e401e2f5d41d5decc4ba3de2fca6bdf2324b4021abbfe6fd2361bdbfd2eb9cc49c8fb619a18a8cfbe8e5b42311ddba4b529e5226723891700070c86da311a58191a84563976bd3c9b7cabc0da5513262c885ba7c628a7795f45d5fd9a99606b45abed9dc66284086b5379789cc112e96b9f65f11df16309c789c14b086a73fd87d5642115b3e39605676af4 -->

ChrisRackauckas commented:
@jClugstor handle this, and stay on top of the bumps:

https://github.com/SciML/OrdinaryDiffEq.jl/blob/master/lib/OrdinaryDiffEqNonlinearSolve/Project.toml
https://github.com/SciML/OrdinaryDiffEq.jl/blob/master/lib/OrdinaryDiffEqRosenbrock/Project.toml
https://github.com/SciML/OrdinaryDiffEq.jl/blob/master/lib/OrdinaryDiffEqFIRK/Project.toml
https://github.com/SciML/OrdinaryDiffEq.jl/blob/master/lib/OrdinaryDiffEqBDF/Project.toml

Ping @oscardssmith

JuliaRegistrator committed:
New version: OrdinaryDiffEqDifferentiation v1.5.0

UUID: 4302a76b-040a-498a-8c04-15b101fed76b
Repo: https://github.com/SciML/OrdinaryDiffEq.jl.git
Tree: 8128445fe4b120cdeabe2fe25ed2215f72587ff0

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
